app/src/main/AndroidManifest.xml

```diff
@@ -108,7 +108,7 @@
 		</activity>
 		<activity
 			android:name=".activity.OoniRunActivity"
-			android:label="@string/APP_NAME"
+			android:label=""
```

Comment:
I would like to know why we don't need this label anymore

Reply:
This displays at the title of the page. Adding it will mean we can no longer conform to the designs